setup.cfg

```diff
@@ -48,7 +48,7 @@ files =
 
 
 [pep257]
-ignore = D102,D104,D203,D105,D213
+ignore = D102,D107,D104,D203,D105,D205,D212,D213,D401,D412,D406,D407,D411,D413,D417
```

Comment:
are the failures too much to fix?

Reply:
I have to dig it into these issues but it seems to be only formatting issues, I can have a look.

Reply:
I fixed most of the rules, the other ones require some doc rewrite that can maybe handled later ?